README.rst

```diff
@@ -0,0 +1,183 @@
+
+Q: Once the next version of Ansible is released, will my playbooks still work
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For users of the community version of ansible pip/apt-get install ansible will continue to give you a working install of Ansible including the three thousand plus modules.
+
+Q: I deploy Ansible directly from devel. Is that still advised?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We recognize that many users install Ansible directly from devel, and generally we do our best to keep the devel branch relatively stable. These changes, however, will have a very large short-term impact, and we cannot guarantee that devel will be as stable as it has been in the past through this migration process.  
+
+Q: When will the next version of Ansible be released with these changes?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We don't have a firm date yet, but we plan to release Ansible 2.10 sometime in 2020, and we do expect to have several alpha/beta releases between now and then. Until that time, Ansible 2.9 will continue to be the supported version.
+
+Q: I'd like to consume the development stream of Ansible during this process. How do I do that?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once the split has been successfully accomplished, devel users will consume ansible-base and collections separately. We will provide instructions for installing the various components from devel when it is unfrozen on March 23.
+
+Contributors to Ansible
+------------------------
+
+`Using Developing Collections <https://docs.ansible.com/ansible/latest/dev_guide/developing_collections.html>`_
+
+Q: I'd like to submit a new module or plugin to Ansible. How shall I proceed?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you're a vendor/partner and you're writing Ansible content to interact with your software, we recommend writing your own collection. This will allow you to pursue certification against the Ansible Automation Platform. For more info on certification, read here [FIXME: link].
+
+If you want to submit your module to the General Community Collection, you can find those guidelines here [FIXME: link].
+
+If you want to submit your module to an existing collection, you'll want to coordinate with the maintainers of those collections and follow their guidelines. Note that not all collections will necessarily accept new modules. 
+
+Ansible Base will no longer accept new modules.
+
+Q: I'd like to submit a pull request to an existing Ansible module. How will I know where this module will live?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We will have a mapping of old modules to their new homes. Should you submit a PR to the wrong repository, we will close it and point you to the correct repository.
```

Comment:
Is there already a mapping? It could be good to make it polished enough people could reference it on their own—otherwise people are going to be filing tons of issues against ansible/ansible anyways.

Reply:
botmeta has new field 'migrated_to' which will have collection name from galaxy